lib/scavenger_hunt/app/assets/stylesheets/scavenger_hunt/components/_helper.sass

```diff
@@ -9,3 +9,13 @@ a.no-border
 .end-game
   color: $text
   border-color: $text !important
+
+.game-progress
+  padding: 3px
+
+.secondary-footer
```

Comment:
First off, I need some guidance with naming convention here. The idea here is that on most of the pages there is the standard game footer with "end game", "see all clues", "time", and "completed". Then, above that, there is often another div of elements that, in the design guide, sit just above it. So, I needed a way to pull those things down like, "return to clue" and "next hint" etc. If you can think of a better way to do this or if there is a file this would fit better in, I can move it! 

The second thing about this is the code is almost the same as toolbar--bottom, except the `bottom` is different and there is no `background` set on it. I tried to use @extend, but couldn't get it to work. Do you have any thoughts on how/if I should somehoe relate this to `toolbar--bottom`? @flipsasser

Reply:
I'll admit that when I wrote these styles, I struggled mightily with the different type of toolbars. I have no qualms with how you're pursuing it here!

Reply:
Have you tested this in a mobile browser? `position: fixed` is a real gotcha in those cases!

Reply:
@flipsasser Just plugged in my phone and tested the Scavenger Hunt and it all looks good!